lib/src/screens/exchange/exchange_template_page.dart

```diff
@@ -150,9 +150,8 @@ class ExchangeTemplatePage extends BasePage {
                               .color,
                           currencyValueValidator: AmountValidator(
                               type: exchangeViewModel.wallet.type),
-                          addressTextFieldValidator: AddressValidator(
-                              type:
-                              exchangeViewModel.depositCurrency),
+                          //addressTextFieldValidator: AddressValidator(
```

Comment:
Better to make new validator which will use address validator and domain validator (create if not exists) and keep validation for address fields.